SLACK.md

```diff
@@ -0,0 +1,30 @@
+# OPA Slack Guidelines v1.0
+
+The [CNCF Code of
+Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+applies in all areas in addition to these rules:
+
+## Channel Spamming
+
+Outside of the issue queues, this is our best communication medium for projects.
```

Comment:
Do we want to replace `this` with `Slack`? I found it a little confusing trying to piece together what "this" was in this context.

Reply:
Done.